spec/features/ctc/eitc_spec.rb

```diff
@@ -42,6 +42,24 @@
     expect(page).not_to have_text 'lumen inc'
   end
 
+  scenario "a client who starts a W-2 can exit the app then continue from the portal" do
+    fill_in_can_use_ctc(filing_status: "single", claim_eitc: true)
+    fill_in_eligibility
+    fill_in_basic_info
+
+    expect(page).to have_selector("h1", text: I18n.t('views.ctc.questions.investment_income.title',current_tax_year: current_tax_year))
```

Comment:
For some reason this line is failing. Maybe the order of quetsions moved around?